src/components/NotificationContainer.tsx

```diff
@@ -139,19 +149,49 @@ function NotificationsContainer() {
     lastShownNotification.current = latestNotificationItem
   }
 
-  return (
+  const showDialog = React.useCallback(() => {
+    setDialogOpen(true)
+  }, [setDialogOpen])
+
+  const closeDialog = React.useCallback(() => {
+    closeNotification()
+    setDialogOpen(false)
+  }, [setDialogOpen, closeNotification])
+
+  const dialog = (
+    <Dialog fullScreen open={dialogOpen} onClose={closeDialog} TransitionComponent={FullscreenDialogTransition}>
+      <DialogBody
+        top={<MainTitle onBack={closeDialog} title={"Error"} />}
+        actions={
+          <DialogActionsBox>
+            <ActionButton autoFocus icon={<CheckIcon />} onClick={closeDialog} type="primary">
+              {"Dismiss"}
+            </ActionButton>
+          </DialogActionsBox>
+        }
+      >
+        <Box alignSelf="center" margin="24px auto 0" maxWidth={400} width="100%">
+          <Typography>{notification ? notification.message : ""}</Typography>
+        </Box>
+      </DialogBody>
+    </Dialog>
+  )
+
+  const notificationJSX = (
     <>
       <Notification
         autoHideDuration={5000}
         message={notification ? notification.message : ""}
         type={notification ? notification.type : "error"}
         open={open}
-        onClick={notification ? notification.onClick : undefined}
+        onClick={notification && notification.onClick ? notification.onClick : showDialog}
         onClose={closeNotification}
       />
       <OfflineNotification message="Offline" open={!isOnline} />
     </>
   )
+
+  return dialogOpen ? dialog : notificationJSX
```

Comment:
As a general principle: We shouldn't render dialogs conditionally. This will definitely break the transition animation.